loom/script/native/lsNativeInterface.cpp

```diff
@@ -542,6 +542,8 @@ void lualoom_newnativeuserdata(lua_State *L, NativeTypeBase *nativeType, void *p
     // set the user data's native type
     ud->m_nativeType = nativeType;
 
+    ud->m_owner = owner;
```

Comment:
Oh. So we should probably mention it here too as it's salient to the operation of this particular API call. It seems optimistic to expect someone to trace the flag down to where it's stored to know it's important to use it or what it does?